src/pages/admin/index.tsx

```diff
@@ -91,3 +61,33 @@ export default Admin;
 Admin.getLayout = function getLayout(page: React.ReactElement) {
   return <HomeLayout>{page}</HomeLayout>;
 };
+
+export const getServerSideProps: GetServerSideProps = async ctx => {
+  const { req } = ctx;
+
+  const token = req.cookies['accessToken'];
+
+  if (!token) {
+    return {
+      redirect: {
+        permanent: false,
+        destination: '/',
+      },
+    };
+  } else {
+    //TODO : server 주소 변경
+    const isAdmin = await fetch(`http://localhost:3001/adminCheck?query=${token}`).then(res => res.json());
+    if (!isAdmin) {
```

Comment:
변수로 바꿔주세요! 
그리고 axios 안쓰고 fetch 쓴 이유가 있나요 ?

```suggestion
    const isAdmin = await axios.get(`${API_END_POINT}/adminCheck`,{params:{query:token});
    if (!isAdmin) {
```

Reply:
앗 fetch 사용만 가능한줄알고 fetch 사용했어요!
axios로 수정해서 다시 올리겠습니답